lib/platform/gasops.c

```diff
@@ -47,6 +47,10 @@
 #define gas_reg_write64(dev, val, reg) __gas_write64(dev, val, \
 						     &dev->gas_map->reg)
 
+#define IS_MRPC_CMD_NO_RETRY(x) (((x & 0xfff) > 0x100) && \
+				 ((x & 0xfff) < 0x120) && \
+				 ((x & 0xfff) != 0x107))
```

Comment:
This is still gross. If we are going to operate on ranges of commands we should at least use proper defines that are called out in enum mrpc_cmd....  Not sure why the we need the "& 0xfff" here either as switchtec_cmd() is only called (and only should be called) with values from enum mrpc_cmd.

Though I'd actually probably prefer something more like this:

```
static const char gasop_noretry_cmds[] = {
    [MRPC_GET_DEV_INFO] = 1,
    [MRPC_SECURITY_CONFIG_GET] = 1,
    ....etc
};

static inline bool gasop_is_no_retry_cmd(uint32_t cmd)
{
        if (cmd >= array_size(gasop_noretry_cmds))
                return 0;
         return gasop_noretry_cmds[cmd];
}
```

Reply:
Good idea. Updated.

The 0xfff has been changed to SWITCHTEC_CMD_MASK. It is needed because the PAX ID will be OR-ed into command ID (bit 18 and up).